### DISCUSSION TITLE
"transforms" in dictionary based transforms missing in 0.9.1?

### DISCUSSION THREAD

**Discussion by maxencewynen** [Project-MONAI/MONAI#5114] [General]
Hi all,
I noticed the usual "transforms" keys I had after performing transformations on a dictionary based dataset was missing.
Was it replaced by another feature? How can I access the list of transformations the image/segmentation went through in 0.9.1?
Best
Maxence

**Comment by wyli** [comment]
that is now at MetaTensor's `applied_operations` property, so it's also available for array-based transforms. please see 
https://github.com/Project-MONAI/MONAI/discussions/4764 
https://colab.research.google.com/drive/1T4iAys-cC2qL80oJkIbAXAPlWNPwp4H7

**Reply by maxencewynen** [reply]
Thanks !
